mibidata/color.py

```diff
@@ -224,3 +225,51 @@ def composite(image, color_map, gamma=1/3, min_scaling=10):
             gamma)
     screened = _screen(data_map)
     return np.uint8(screened * 255)
+
+
+def compose_overlay_from_image_data(image, overlay_settings):
+    """Overlays multiple image channels using overlay_settings from mibitracker.
+
+    Args:
+        image: A MibiImage.
+        overlay_settings: Color overlay dictionary with the following fields:
+            'brightness': float between -1 and 1.
+            'channel': Name of the MibiImage channel.
+            'color': One of the following: 'Cyan', 'Yellow', 'Magenta', 'Green',
+                'Orange', 'Violet', 'Red', 'Blue', or 'Gray'.
+            'intensity_higher': Upper limit of the channel intensity.
+            'intensity_lower': Lower limit of the channel intensity.
+
+    Returns:
+        An NxMx3 uint8 array of an RGB image.
+    """
+    # Convolve with this filter no matter what, to mimic browser rendering.
+    kernel = np.array([
+        [0.05, 0.1, 0.05],
+        [0.1, 0.4, 0.1],
+        [0.05, 0.1, 0.05]
+    ])
+    for i, item in enumerate(overlay_settings):
```

Comment:
This does not work directly with the json as downloaded from mibitracker, which has the form:

```
{
  'image_id': {
    'channels': {
      'channel': {'brightness': float, 'color': color, ...}
    }
  }
}
```

You can either make it work with this for any image id with something like 

```
[settings] = overlay_settings.values()
for i, (channel, item) in enumerate(settings['channels'].items()):
  int_array = image[channel]
  ...
```

or expect the user to have selected the the `'channels'` already, in which case you still need to iterate not only through the key.